pkg/registry/views/project-page.html

```diff
@@ -1,20 +1,20 @@
-<filter-bar class="content-filter">
-</filter-bar>
+<div class="content-filter">
+    <div class="listing-actions">
+        <button class="btn btn-danger btn-delete pficon pficon-delete"
+            ng-click="" disabled></button>
+        <button class="btn btn-default pficon pficon-edit"
+            ng-click="" disabled></button>
+    </div>
+    <h3>
+        <i class="pficon pficon-project"></i>
+        {{ project().metadata.name }} / {{ project().metadata.annotations["openshift.io/description"] }}
```

Comment:
I've clarified this here: https://raw.githubusercontent.com/cockpit-project/cockpit-design/master/kubernetes/projects.png